network:
  uploadURL:
    - url: http://127.0.0.1:8081/edgecompute
edgeComputing:
  deviceID: 1
reference_point:
  lng: 114.922222
  lat: 29.687004
radar:
  id: '21'
  hardware_interface: RJ45
  transport_protocol: RX_T500_RJ45
  ip: 192.168.1.3
  port: 17308
  delay_time: 50
  pitch_angle: 2
  north_angle: 3
  rotation_angle: 4
  h: 1
  lx: 5
  ly: 6
camera:
  id: '22'
  hardware_interface: USB
  software_interface: '1'
  transport_protocol: UVC
  u: 2448
  v: 2048
  fx: 2401.437239
  fy: 2499.714017
  cx: 1197.2432
  cy: 992.6659
  k1: 0.2628
  k2: -0.9531
  p1: -0.05358
  p2: 0.002628
  k3: 1.163314
  delay_time: 50
  h: 7000
  pitch_angle: 8
  north_angle: 3
  lx: 0
  ly: 0
